src/device.coffee

```diff
@@ -167,6 +167,8 @@ exports.getDeviceType = do ->
 				return configFromFile.deviceType
 
 do ->
+	APPLY_STATE_SUCCESS_DELAY = 50
```

Comment:
Can we increase this to 1s, .05s is probably too small to have any noticeable effect